### ISSUE TITLE
Evaluate QA models using run_qa.py script

### ISSUE THREAD

**Issue by Yulong-W** [huggingface/transformers#22480]
### System Info

Hello, could anyone demonstrate the command lines to ONLY evaluate a fine-tuned QA model e.g., deberta on the dev set of squad 1.1 (https://raw.githubusercontent.com/rajpurkar/SQuAD-explorer/master/dataset/dev-v1.1.json) using the run_qa.py script? just keep getting errors and feel confused about the explanation of the arguments. Thanks!

### Who can help?

@sgugger 

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [X] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

NA

### Expected behavior

NA

**Comment by sgugger** [comment]
The same command given in the README without the training prompts works perfectly. Just tried:
```
python examples/pytorch/question-answering/run_qa.py \
  --model_name_or_path deepset/deberta-v3-base-squad2 \
  --dataset_name squad \
  --do_eval \
  --max_seq_length 384 \
  --doc_stride 128 \
  --output_dir ~/tmp/debug_squad/
```
And it proceeds without error to an F1 of 92.28 and accuracy of 85% for exact matches.

**Comment by Yulong-W** [comment]
> The same command given in the README without the training prompts works perfectly. Just tried:
> 
> ```
> python examples/pytorch/question-answering/run_qa.py \
>   --model_name_or_path deepset/deberta-v3-base-squad2 \
>   --dataset_name squad \
>   --do_eval \
>   --max_seq_length 384 \
>   --doc_stride 128 \
>   --output_dir ~/tmp/debug_squad/
> ```
> 
> And it proceeds without error to an F1 of 92.28 and accuracy of 85% for exact matches.

Thanks Sylvain! Yes, these commands work well, but what I am wanting to do is to just evaluate a model on my own test set (my own test set follows the same format as the original SQuAD dataset), and I would appreciate if you could provide suggestion on how to modify those commands.

**Comment by sgugger** [comment]
You will need to adapt the example (which is just that, an example) to the format of your own data.

**Comment by Yulong-W** [comment]
> You will need to adapt the example (which is just that, an example) to the format of your own data.

Thanks but by saying "adapt the example", do you mean adapt the commands or the script? Does that mean I do not need to change the format of my test set (i.e., the SQuAD json is okay)?
